src/Components/Artist/MarketInsights.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface MarketInsightsProps extends RelayProps, React.HTMLProps<MarketInsights> {}
+
+const Categories = {
+  "blue-chip": "Blue Chip",
+  "top-established": "Top Established",
+  "top-emerging": "Top Emerging",
+}
+
+export class MarketInsights extends React.Component<MarketInsightsProps, null> {
+  renderGalleryRepresentation() {
+    const { highlights } = this.props.artist
+    const { partners } = highlights
+    if (partners && partners.edges && partners.edges.length > 0) {
+      const partner = partners.edges[0].node
+      const { categories } = partner
+      let category
+      Object.keys(Categories).forEach(key => {
+        categories.forEach(partnerCategory => {
+          if (partnerCategory.id === key) {
+            category = Categories[key]
+          }
+        })
+      })
```

Comment:
There’s also a `find` method on arrays.

Reply:
Ah but that may not be available when iterating over objects such as `Categories.